tests/aws/services/cloudformation/resources/test_lambda.py

```diff
@@ -95,6 +95,35 @@ def test_lambda_w_dynamodb_event_filter_update(deploy_cfn_template, snapshot, aw
     snapshot.match("updated_source_mappings", source_mappings)
 
 
+@pytest.mark.skipif(
```

Comment:
Was there supposed to be a condition on it? Otherwise I'd suggest simplifying to `skip`
```suggestion
@pytest.mark.skip(
```